### ISSUE TITLE
weird validation message

### ISSUE THREAD

**Issue by ousia** [veraPDF/veraPDF-arlington-tools#1]
First of all, sorry if this is not the right repository to report issues.

I’m trying to implement `/GoToE` links in PDF documents.

My [sample PDF document](https://github.com/veraPDF/veraPDF-arlington-tools/files/12848898/uncompressed-a.pdf) contains these two objects:

``` pdf
5 0 obj
<<
  /D [ 0 /Fit ]
  /NewWindow true
  /S /GoToE
  /T <<
    /N (embedded-document.pdf)
    /R /C
  >>
>> 
endobj 

%[…]

28 0 obj
<<
  /Names [ (embedded-document.pdf) 2 0 R ]
>>
endobj
```

When the target dictionary contains a child document, `/N` must be the name of the file in the `/EmbeddedFiles` name tree.

The error I get is the following:

```
Entry N with type StringByte in Target does not satisfy special case:

  (fn:Eval(fn:Not(fn:IsPresent((@R == C) && fn:InNameTree(trailer::Catalog::Names::EmbeddedFiles)))))
```

I must admit I’m not familiar with the syntax, but I would say that the above may read as the opposite of the statement above (not that name in `/N` with child relation and name in embedded files name tree).

Otherwise, I don’t understand what this `/GoToE` is doing wrong.

Many thanks for your help.

**Comment by MaximPlusov** [comment]
hello @ousia. The Arlington contains both requirements:
- `fn:IsRequired((@R==C) && fn:InNameTree(trailer::Catalog::Names::EmbeddedFiles))` in the column `Required`
- `fn:Eval(fn:Not(fn:IsPresent((@R==C) && fn:InNameTree(trailer::Catalog::Names::EmbeddedFiles))))` in the column `SpecialCase`
It looks like the second requirement should be the opposite. I created [PR](https://github.com/pdf-association/arlington-pdf-model/pull/102) fixing it to the Arlington pdf model.

**Comment by MaximPlusov** [comment]
Fixed in the latest dev build 1.25.97.
